### PR TITLE
TOOLS-150 TOOLS-151 ize secret

### PR DESCRIPTION
**Pull request by psihachina**
Changelog: 
- rename ize config to ize secret
- mark flags as required

## Test: 
[asciicast](https://asciinema.org/a/ZNcG1avQsI7R9pKxsE3tncyYY)

### CODE REVIEW

File: internal/commands/secret.go [hunk 1]
```diff
@@ -112,9 +112,13 @@ func (b *commandsBuilder) newConfigCmd() *configCmd {
 
 	removeCmd.Flags().StringVar(&cc.vaultType, "type", "", "vault type")
 	removeCmd.Flags().StringVar(&cc.path, "path", "", "path to secrets")
+	removeCmd.MarkFlagRequired("type")
+	removeCmd.MarkFlagRequired("path")
 
 	setCmd.Flags().StringVar(&cc.vaultType, "type", "", "vault type")
 	setCmd.Flags().StringVar(&cc.filePath, "file", "", "file with sercrets")
+	setCmd.MarkFlagRequired("type")
```
Comment (AutomationD) [inline internal/commands/secret.go:120]: Can we also print the usage of the sub command after the error?

**Review by AutomationD — CHANGES_REQUESTED** [review]
Looks good, but I have a question

**Review by igorkotof — APPROVED** [review]

File: internal/commands/secret.go (hunk 1 above)
Comment (AutomationD) [reply internal/commands/secret.go:120]: Also, if talking idealistic way, `ParameterAlreadyExists` error doesn't make sense when we deal with `secrets`. Maybe we could catch the error and use `Secret already exists, please use `--overwrite` or something like that?

(also maybe we should rename `--overwrite` to `--force`?)

Comment (psihachina) [reply internal/commands/secret.go:120]: Yes, by default viper after every error can print a help message, but the error maybe is not be syntax. I see strange this when an error in logic and app print help message. If you want to use this feature with print help message I see the way when we do not send logic errors up and handle them in function command. Thus we do not print help messages when catching a logic error

Comment (psihachina) [reply internal/commands/secret.go:120]: `ParameterAlreadyExists ` is a standard error in AWS SDK

Comment (AutomationD) [reply internal/commands/secret.go:120]: So can we catch the Parameter Store error easily? as it's a bit confusing for the user as they don't really know what a "parameter" as they are dealing with "Secrets". If we can't do it easily, let's skip this for now.

Comment (psihachina) [reply internal/commands/secret.go:120]: Yes, we can catch this error - it's easy, and we can also just overwrite secrets by setting the overwrite parameter to true.

**Review by AutomationD — APPROVED** [review]

**Pull request merged**